compose/snippets/src/main/java/com/example/compose/snippets/touchinput/focus/FocusSnippets.kt

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+@file:Suppress("DEPRECATION_ERROR")
```

Comment:
same as above

Reply:
Same comment here. I think there may be other snippets we'll have to update once 1.7.0 goes stable (like if we deprecate other APIs). 

Perhaps it'd be helpful for me to mention our branch policy in the README so that it is clear?